Modify and add some .qmd and .R files.

Kwan-Jenny opened the pull request:
I modified two .qmd files and added one new .qmd file. I also added some R function files. I think I may need to delete some unnecessary R function files.

There are three .qmd files. Please review them step by step:
1) shigella-kinetics.qmd
2) _sec_outtakes-data-load.qmd
3) simulation-shigella.qmd

Please go through the .qmd files above in order.

Kwan-Jenny committed:
# Edit first part

Kwan-Jenny committed:
# second update

Kwan-Jenny committed:
# third update

d-morrison reviewed (changes requested):
This PR is looking great! Please address my [inline comments](https://github.com/UCD-SERG/shigella/pull/4/files), and try to get all the [automated checks](https://github.com/UCD-SERG/shigella/pull/4/checks) to pass; if you have questions about any of those checks, please tag me in a comment on this PR, and we can discuss them when we meet.

Kwan-Jenny committed:
Update R/calculate_metrics.R

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update R/generate_final_table.R

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update R/simulate_seroincidence2.R

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/_sec_outtakes-data-load.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/shigella-kinetics.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/_sec_outtakes-data-load.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/simulation-shigella.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/simulation-shigella.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/simulation-shigella.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/simulation-shigella.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
Update vignettes/articles/simulation-shigella.qmd

Co-authored-by: Douglas Ezra Morrison <demorrison@ucdavis.edu>

Kwan-Jenny committed:
# version increment

Kwan-Jenny committed:
update 1

Merge branch 'practice_Kwan' of https://github.com/UCD-SERG/shigella into practice_Kwan
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

Kwan-Jenny committed:
# delete files

Kwan-Jenny committed:
# edit function code in R folder

Kwan-Jenny committed:
# edit basic parts on qmd

Kwan-Jenny committed:
# update NAMESPACE

Kwan-Jenny committed:
# update NEWS file

Kwan-Jenny committed:
update WORDLIST for spellcheck
